gridcomps/History/MAPL_HistoryGridComp.F90

```diff
@@ -3543,6 +3543,7 @@ subroutine Run ( gc, import, export, clock, rc )
                list(n)%unit = -1
             end if
          elseif (list(n)%sampler_spec == 'station') then
+            call MAPL_TimerOn(GENSTATE,"Station_preRun")
```

Comment:
This timer appears to be timing a lgr output and two assignments.  That seems unnecessary.  Please eliminate unless I've misunderstood.